src/helpers/fetchHandlers.ts

```diff
@@ -0,0 +1,40 @@
+import MiscService from '../services/misc';
+
+export default class FetchHandler {
+  private static sourceHandlerMap: Map<string, any> = new Map();
+  private static destHandlerMap: Map<string, any> = new Map();
+  private static deletionHandlerMap: Map<string, any> = new Map();
+
+  public static getDestHandler(dest: string, version: string) {
+    let destinationHandler: any;
+    if (this.destHandlerMap.get(dest)) {
```

Comment:
if(this.destHandlerMap.has(dest)) change all the place where this pattern is relevant

Reply:
this is handled